css/style.css

```diff
@@ -0,0 +1,166 @@
+body {
+  width: 375px;
+  margin: 0;
+}
+
+header {
+  background-color: #3c3a39;
+}
+
+nav {
+  height: 48px;
+  width: 375px;
```

Comment:
- [ ] I notice that you are using a fixed width for most of your elements, Which is why your design is not responsive. I advise that you remove all the fixed widths `width: 375px;` you have in your design. Allow the design scale according to screen size. 

#### When I view your design in mobile view. It becomes very tiny.  
![image](https://user-images.githubusercontent.com/84156714/211929528-e915a3b6-caff-4f7b-bca1-fc32b00bcde7.png)